etc/profile-a-l/electron.profile

```diff
@@ -22,7 +22,7 @@ include whitelist-runuser-common.inc
 include whitelist-usr-share-common.inc
 include whitelist-var-common.inc
 
-# Add the next line to your chromium-common.local if your kernel allows unprivileged userns clone.
+# Add the next line to your electron.local if your kernel allows unprivileged users clone.
```

Comment:
You can not clone users.

Reply:
To clarify, "userns" here means user namespaces.

See also namespaces(7) and the `userns` option in firejail.config:

```
# Enable or disable user namespace support, default enabled.
# userns yes
```